sos/utilities.py

```diff
@@ -174,7 +169,7 @@ def _child_prep_fn():
             raise e
 
     if p.returncode == 126 or p.returncode == 127:
-        stdout = six.binary_type(b"")
+        stdout = bytes
```

Comment:
Should this be `stdout = b""`? That way we're still returning a empty string-like rather than the class.

```python
>>> stdout = six.binary_type(b"")
>>> stdout
b''
>>> stdout = bytes
>>> stdout
<class 'bytes'>
>>> stdout = b""
>>> stdout
b''

```

Reply:
Indeed - and that was the test that I wasn't sure about.  Oops!